Content.Shared/Movement/Components/MovementSpeedModifierComponent.cs

```diff
@@ -15,8 +15,8 @@ public sealed partial class MovementSpeedModifierComponent : Component
 
         // weightless
         public const float DefaultWeightlessFriction = 1f;
-        public const float DefaultWeightlessModifier = 0.7f;
-        public const float DefaultWeightlessAcceleration = 1f;
+        public const float DefaultWeightlessModifier = 0.75f;
+        public const float DefaultWeightlessAcceleration = 1.25f;
```

Comment:
This would change it for everyone, not just borgs. Revert and put it in the YAML file `base_borg_chasis.yml` (I put it in the suggestions already so just revert this line).



```suggestion
        public const float DefaultWeightlessModifier = 0.70f;
        public const float DefaultWeightlessAcceleration = 1f;
```